backend/db.js

```diff
@@ -0,0 +1,44 @@
+export const PRODUCTS = [
```

Comment:
It is recomment to write normal variables in camelcase, so you should write `products` instead of `PRODUCTS`.